packages/gatsby-source-faker/README.md

```diff
@@ -1,37 +1,37 @@
-## gatsby-source-faker
-
-This is a plugin that allows you to use [faker.js](https://github.com/marak/Faker.js/) to generate fake data for gatsby sites. This could come in handy for creating example sites, documentation, or just to experiment with Gatsby.js
-
-### To use it
-
-Install `gatsby-source-faker`
-
-```
-    npm install --save gatsby-source-faker
-```
-
-or
-
-```
-    npm install gatsby-source-faker
-```
-
-Add `gatsby-source-faker` to the `gatsby-config.js` as follows
-
-```javascript
-plugins: [
-  {
-    resolve: `gatsby-source-faker`,
-    // derive schema from faker's options
-    options: {
-      schema: {
-        name: ["firstName", "lastName"],
-      },
-      count: 3, // how many fake objects you need
-      type: "NameData", // Name of the graphql query node
-    },
-  },
-]
-```
-
-Example: [Using Faker](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-faker)
+## gatsby-source-faker
```

Comment:
I am not honestly sure why this file was updated.

Reply:
Also happened to me locally. Don't know why :(

Reply:
i.e. after this change, right? Or you mean it happened to you earlier?

Reply:
Earlier, when I checked out different branches (don't know the exact steps to reproduce it)

Reply:
I ran `dos2unix` on this file so _maybe_ it's fixed now 🤞 

But again, I don't really know the issue so 🙃